lib/rubocop/cop/layout/case_indentation.rb

```diff
@@ -109,6 +109,19 @@ module Layout
       #   else
       #     y / 3
       #   end
+      #
+      #   # good - `else` or `when` or `in` and `end`  same line
+      #   case n
+      #   when 0
+      #   else end
+      #
+      #   case n
+      #   when 0
+      #   when 1 then end
+      #
+      #   case n
+      #   in pattern1 then 0
+      #   in pattern2 then 1 end
```

Comment:
Hm, I'm worried whether it's preferable to show these as "good" style examples. These may not be needed as examples.

Reply:
I thought that was certainly true. I think it was quite excessive.
I've removed the relevant "good examples".